src/Symfony/Component/Console/Application.php

```diff
@@ -768,20 +768,39 @@ public static function getAbbreviations($names)
 
     /**
      * Renders a caught exception.
+     *
+     * @deprecated since Symfony 4.4, use "renderThrowable()" instead
      */
     public function renderException(\Exception $e, OutputInterface $output)
+    {
+        @trigger_error(sprintf('The "%s::renderException()" method is deprecated since Symfony 4.4, use "renderThrowable()" instead.', \get_class($this)));
```

Comment:
To me it looks like those deprecations are actually BC breaks the way it's implemented. If someone overwrote `renderException`, it won't work anymore because the method is not called at all.

Reply:
Should be good now.